static/app/components/dropdownMenuDeprecated.tsx

```diff
@@ -111,6 +111,17 @@ type State = {
   isOpen: boolean;
 };
 
+/**
+ *
+ * Deprecated dropdown menu. Use these alternatives instead:
+ * - For a select menu: `CompactSelect`.
+ * - For an action menu: `DropdownMenuControl`.
```

Comment:
I would maybe link to storybook stories for these components. Also does our storybook explain the difference between "select" and "action"?

Reply:
It doesn't yet – currently on my to do list.